add explicit targetless (#169)
metalbear-co/mirrord.dev: content/docs/guides/targetless/index.md

mirrord exec /bin/my-tool
```

### IDE

> If you want to run in targetless mode on IntelliJ or VSCode, you can add

Reviewer: ```suggestion
If you want to run in targetless mode using the IntelliJ or VSCode plugin, you can select the `No Target ("targetless")` option from the target selection dialog, or you can add
```

> If you want to run targetless on IntelliJ or VSCode via mirrord config file, you can add

Reviewer: ```suggestion
If you want to run in targetless mode on IntelliJ or VSCode, you can add
```

```json
{
  "target": "targetless"
}
```

> to your configuration file.

Reviewer: ```suggestion
to your mirrord configuration file.
```
